src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement/appsettings.Development.json

```diff
@@ -29,6 +42,9 @@
     },
     "ResourceRegister": {
       "Endpoint": "https://platform.at22.altinn.cloud"
+    },
+    "AltinnRole": {
+      "Endpoint": "https://at22.altinn.cloud"
```

Comment:
Hvor skal denne verdien egentlig konfigureres opp for de forskjellige miljøene

Reply:
Skrevet om til å bruke det eksisterende SBLBridge endepunktet som brukes fra Register så dette er ikke lenger relevant